Firestore/Source/API/FIRWriteBatch.m

```diff
@@ -93,9 +93,13 @@ - (FIRWriteBatch *)deleteDocument:(FIRDocumentReference *)document {
   return self;
 }
 
-- (void)commitWithCompletion:(void (^)(NSError *_Nullable error))completion {
+- (void)commit {
```

Comment:
This implementation doesn't seem right: it doesn't actually write any mutations.

I would expect this to be implemented as

```
[self commitWithCompletion:nil];
```

Reply:
done